packages/stash/src/actions/deleteRecord.ts

```diff
@@ -0,0 +1,37 @@
+import { Table } from "@latticexyz/config";
+import { Key, Stash } from "../common";
+import { encodeKey } from "./encodeKey";
+import { registerTable } from "./registerTable";
+
+export type DeleteRecordArgs<table extends Table = Table> = {
+  stash: Stash;
+  table: table;
+  key: Key<table>;
+};
+
+export type DeleteRecordResult = void;
+
+export function deleteRecord<table extends Table>({ stash, table, key }: DeleteRecordArgs<table>): DeleteRecordResult {
+  const { namespaceLabel, label } = table;
+
+  if (stash.get().config[namespaceLabel] == null) {
```

Comment:
ooc any advantage of using `stash.get()` over `stash._.get`?

Reply:
yeah exactly, trying to only use `._.state` when necessary, which is only when needing to write